packages/checkout/sdk-sample-app/src/components/SmartCheckoutForm.tsx

```diff
@@ -53,7 +53,7 @@ export const SmartCheckoutForm = ({ checkout, provider }: SmartCheckoutProps) =>
   const [id, setId] = useState<string>('');
   const [idError, setIdError] = useState<string>('');
   const [tokenAddress, setContractAddress] = useState<string>('');
-  const [contractAddressError, setContractAddressError] = useState<string>('');
```

Comment:
seeing some errors in this file still as theres some `setContractAddressError` and `contractAddressError` references in the code